.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
     # run tests, submit coverage info
     - COVERAGE="true"
     # pylint checks skip the tests
-    - RUN_PYLINT="true" SKIP_TESTS="true"
+    - RUN_PYLINT="true" RUN_MYPY="true" SKIP_TESTS="true"
```

Comment:
I'd recommend just having another item in the matrix, so it's more obvious from the main travis page what failed.